doc/start/get-involved.rst

```diff
@@ -23,12 +23,13 @@ These are exciting times in the Ceph community! Get involved!
 +----------------------+-------------------------------------------------+-----------------------------------------------+
 | **IRC**              | As you delve into Ceph, you may have questions  |                                               |
 |                      | or feedback for the Ceph development team. Ceph | - **Domain:** ``irc.oftc.net``                |
-|                      | developers are often available on the ``#ceph`` | - **Channels:** ``#ceph`` and ``#ceph-devel`` |
-|                      | IRC channel particularly during daytime hours   |                                               |
-|                      | in the US Pacific Standard Time zone.           |                                               |
+|                      | developers are often available on the ``#ceph`` | - **Channels:** ``#ceph``                     |
```

Comment:
Consider changing the list of IRC channels to a bullet point list. Otherwise the rendering looks somewhat weird:

![screenshot-2018-6-21 get involved in the ceph community ceph documentation](https://user-images.githubusercontent.com/263427/41729175-d5528e58-7578-11e8-8a94-6adf7057dd9b.png)